packages/ui/src/utils/setUserAgent/constants.ts

```diff
@@ -1,24 +1,66 @@
 import {
+  AuthUserAgentInput,
+  GeoUserAgentInput,
+  InAppMessagingUserAgentInput,
+  StorageUserAgentInput,
   setCustomUserAgent,
   SetCustomUserAgentInput,
   AuthAction,
+  GeoAction,
+  InAppMessagingAction,
+  StorageAction,
   Category,
 } from '@aws-amplify/core/internals/utils';
 
 // replace type once Category input types are available
 export const ACCOUNT_SETTINGS_INPUT_BASE: Omit<
-  SetCustomUserAgentInput,
+  AuthUserAgentInput,
   'additionalDetails'
 > = {
   apis: [AuthAction.DeleteUser, AuthAction.UpdatePassword],
   category: Category.Auth,
 };
 
-// replace type once Category input types are available
-export const IN_APP_MESSAGING_INPUT_BASE: Omit<
+export const AUTHENTICATOR_INPUT_BASE: Omit<
   SetCustomUserAgentInput,
   'additionalDetails'
 > = {
-  apis: undefined, // TODO
+  apis: [
+    AuthAction.SignUp,
+    AuthAction.ConfirmSignUp,
+    AuthAction.ResendSignUpCode,
+    AuthAction.SignIn,
+    AuthAction.FetchMFAPreference,
+    AuthAction.UpdateMFAPreference,
+    AuthAction.SetUpTOTP,
+    AuthAction.VerifyTOTPSetup,
+    AuthAction.ConfirmSignIn,
+    AuthAction.FetchUserAttributes,
+    AuthAction.SignOut,
+    AuthAction.ResetPassword,
+    AuthAction.ConfirmResetPassword,
+    AuthAction.FederatedSignIn,
+  ],
+  category: Category.Auth,
+};
+
+export const GEO_INPUT_BASE: Omit<GeoUserAgentInput, 'additionalDetails'> = {
+  category: Category.Geo,
+  apis: [GeoAction.SearchByText],
+};
+
+export const IN_APP_MESSAGING_INPUT_BASE: Omit<
+  InAppMessagingUserAgentInput,
+  'additionalDetails'
+> = {
+  apis: [InAppMessagingAction.SyncMessages],
```

Comment:
We shouldn't include syncMessages since we don't call it from our code. The APIs for in-app are the message interaction APIs